### PR TITLE
modules/nixos/common: optimise zram

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: modules/nixos/common/default.nix [hunk 1]
```diff
@@ -40,6 +40,15 @@
 
   zramSwap.enable = true;
 
+  # https://github.com/NixOS/nixpkgs/pull/268121
+  # https://wiki.archlinux.org/title/Zram#Optimizing_swap_on_zram
+  boot.kernel.sysctl = {
+    "vm.page-cluster" = 0;
```
Comment (Mic92) [inline modules/nixos/common/default.nix:46]: Are we using zstd in our case? The linked pull request does set this value conditionally.

Comment (zowoq) [reply modules/nixos/common/default.nix:46]: Yes, zstd is the default.

**Review by Mic92 — APPROVED** [review]
Don't have personal experience with these defaults but the linked sources look legit.

**Comment by sedlund** [comment]
https://github.com/LeSuisse/nixpkgs/blob/afd3a57b83977a2e0fe8c4685ca2016a12103d03/pkgs/os-specific/linux/kernel/common-config.nix#L879

**Pull request merged**